commons/util/src/main/java/io/github/microcks/util/openapi/OpenAPISchemaValidator.java

```diff
@@ -38,287 +38,339 @@
 import java.util.List;
 
 /**
- * Helper class for validating Json objects against their OpenAPI schema. Supported version
- * of OpenAPI schema is https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.1.md.
+ * Helper class for validating Json objects against their OpenAPI schema.
+ * Supported version
+ * of OpenAPI schema is
+ * https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.1.md.
+ *
  * @author laurent
  */
 public class OpenAPISchemaValidator {
 
-   /** A commons logger for diagnostic messages. */
-   private static Logger log = LoggerFactory.getLogger(OpenAPISchemaValidator.class);
+  /** A commons logger for diagnostic messages. */
+  private static Logger log = LoggerFactory.getLogger(OpenAPISchemaValidator.class);
```

Comment:
File was indented with 2 spaces instead of 3. Can you please fix this to avoid noise?